src/term/mod.rs

```diff
@@ -1055,6 +1042,7 @@ impl Term {
         self.tabs = IndexRange::from(Column(0)..self.grid.num_cols())
             .map(|i| (*i as usize) % TAB_SPACES == 0)
             .collect::<Vec<bool>>();
+<<<<<<< HEAD
```

Comment:
Merge conflict :(

(Just pulled and tried to build locally to test)

Reply:
(Thanks for all your work on this though! It's a feature I've been wanting)

Reply:
Oh whoops yes I realised that right away but forgot to push the fix.. done now

Reply:
Thanks. Out of interest (not too hot on github flow) - if wanted to fix it would the correct thing be to fork off of your branch and open another pull request?

Reply:
@tshakah Yes - to be clear though the PR should be opened against neon64's (forked) repo, not this one. That way you can get your changes merged into this branch, and they'll show up here in the 'main' repo automatically; otherwise you'd essentially replace this PR by including all neon64's commits and adding to them. :slightly_smiling_face: